plugins/inputs/airthings/sample.conf

```diff
@@ -0,0 +1,33 @@
+# Read stats from aerospike server(s)
+[[inputs.airthings]]
+  ## Airthings servers to connect to (with port)
+  servers = ["localhost:3000"]
+
+ClientId:     "clientid",
+ClientSecret: "clientsecret",
+TokenUrl:     ts.URL + "/v1/token",
+Scopes:       []string{"read:device:current_values"},
```

Comment:
I would rather live in a world with peace, flowers and consistently indented TOML files

Reply:
Great catch, thanks! Fixed in the latest push.